tests/nightly/gpu/test_transresnet.py

```diff
@@ -8,6 +8,7 @@
 import unittest
 import parlai.scripts.display_data as display_data
 import parlai.core.testing_utils as testing_utils
+from parlai.core.logging_utils import logger
```

Comment:
yeah let's just get rid of all the extra unnecessay logger imports

Reply:
Why isn't the linter complaining about these actually?

Reply:
Not sure about that

Reply:
Just figured this out. Capture output uses the logger object, so whichever file calls capture output needs to have the logger object imported there. For now removed all imports and restored capture output to its original form (can be dealt in the next PR where print is switched to logging)